### PR TITLE
feat: sync translations

### PR DESCRIPTION
**Pull request by michelvermeer**
TODO:

- Read locales from source repository from ...  .env (?)
- Read POEDITOR_API_TOKEN and POEDITOR_PROJECT_ID from github params

### CODE REVIEW

File: packages/sync-translations/package.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  "main": "dist/elseu-sdu-react-scripts-sync-translations.cjs.js",
+  "author": "Michel Vermeer",
+  "version": "1.0.0",
+  "license": "MIT",
+  "files": [
+    "dist"
+  ],
+  "bin": {
+    "sdu-react-scripts-update-deps": "./bin/sdu-sync-translations.js",
+    "sdu-deps": "./bin/sdu-sync-translations.js"
```
Comment (daphnesmit) [inline packages/sync-translations/package.json:13]: Ehhh

File: packages/sync-translations/package.json [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  "description": "Helper package to sync translations of all locales with POEditor",
+  "main": "dist/elseu-sdu-react-scripts-sync-translations.cjs.js",
+  "author": "Michel Vermeer",
+  "version": "1.0.0",
+  "license": "MIT",
+  "files": [
+    "dist"
+  ],
+  "bin": {
+    "sdu-react-scripts-update-deps": "./bin/sdu-sync-translations.js",
```
Comment (daphnesmit) [inline packages/sync-translations/package.json:12]: Wat is dit?

File: packages/sync-translations/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+import fetch from 'node-fetch';
+import path from 'path';
+import PO from 'pofile';
+import request from 'request';
+
+import type { POEditorLanguagesResult, POEditorResponse, POFile } from './types';
+
+const constants = {
+  POEDITOR_PROJECT_ID: '670969',
+  POEDITOR_API_TOKEN: 'fccf4f817d0d7e1f9f957ea8c6d1294e',
```
Comment (daphnesmit) [inline packages/sync-translations/src/index.ts:17]: Hardcoded in repo?

**Review by daphnesmit — CHANGES_REQUESTED** [review]
Security

File: packages/sync-translations/package.json (hunk 1 above)
Comment (michelvermeer) [reply packages/sync-translations/package.json]: Sloppy copy-paste :)

**Comment by skarnl** [comment]
Will close this PR - to get rid of the api-key ... I hope

**Pull request closed without merging**